Standards/scs-0100-v1-flavor-naming.md

```diff
@@ -99,13 +100,15 @@ on L1TF susceptible CPUs w/o effective core scheduling or disabled protections o
 
 #### Examples
 
+```txt
 - SCS-**2C**:4:10n
 - SCS-**2T**:4:10n
 - SCS-**2V**:4:10n
 - SCS-**2L**:4:10n
 - SCS-**2Li**:4:10n
 - ~~SCS-**2**:\*\*4:10n~~ <- CPU suffix missing
 - ~~SCS-**2iT**:4:10n~~ <- This order is forbidden
+```
```

Comment:
I think I made a mistake here -- the underscores are part of the name whereas the double asterisks are meant for emphasis. So the code block doesn't work. Maybe we need to escape all underscores?